app/api/schema/attendees.py

```diff
@@ -31,26 +31,26 @@ def validate_json(self, data, original_data):
     id = fields.Str(dump_only=True)
     firstname = fields.Str(required=True)
     lastname = fields.Str(required=True)
-    email = fields.Str(allow_none=True)
-    address = fields.Str(allow_none=True)
-    city = fields.Str(allow_none=True)
-    state = fields.Str(allow_none=True)
-    country = fields.Str(allow_none=True)
-    job_title = fields.Str(allow_none=True)
-    phone = fields.Str(allow_none=True)
-    tax_business_info = fields.Str(allow_none=True)
-    billing_address = fields.Str(allow_none=True)
-    home_address = fields.Str(allow_none=True)
-    shipping_address = fields.Str(allow_none=True)
-    company = fields.Str(allow_none=True)
-    work_address = fields.Str(allow_none=True)
-    work_phone = fields.Str(allow_none=True)
+    email = fields.Str(allow_none=True, default="Not Specified")
+    address = fields.Str(allow_none=True, default="Not Specified")
+    city = fields.Str(allow_none=True, default="Not Specified")
+    state = fields.Str(allow_none=True, default="Not Specified")
+    country = fields.Str(allow_none=True, default="Not Specified")
+    job_title = fields.Str(allow_none=True, default="Not Specified")
+    phone = fields.Str(allow_none=True, default="Not Specified")
+    tax_business_info = fields.Str(allow_none=True, default="Not Specified")
+    billing_address = fields.Str(allow_none=True, default="Not Specified")
+    home_address = fields.Str(allow_none=True, default="Not Specified")
+    shipping_address = fields.Str(allow_none=True, default="Not Specified")
+    company = fields.Str(allow_none=True, default="Not Specified")
+    work_address = fields.Str(allow_none=True, default="Not Specified")
+    work_phone = fields.Str(allow_none=True, default="Not Specified")
```

Comment:
Default is None, which is much better than "Not Specified"

Reply:
Ohh @iamareebjamal I changed it because the issue mentioned that if this change is not required I guess we can close this issue since this might be confusing for other new comers and they might start contributing to this :sweat_smile:

Reply:
It is still applicable to frontend but not server but very low priority, so yes, I will close the issue